src/include/random.h

```diff
@@ -141,6 +153,11 @@ int generate_random_number(MutexT& m, EngineT& e)
 
 } // namespace detail
 
+bool rng_is_seeded() 
+{
+  return detail::rng_is_seeded_state();
+}
```

Comment:
Ok. Looking at the places we're using rand() right now, there don't seem to be many instances where my concern will be an actual issue-- we're typically just getting the occasional random number. I'll go ahead and implement "automagic".